### PR TITLE
Fe listing edit listing

### PR DESCRIPTION
**Pull request by Gonlonge**
Edit listing page OK

### CODE REVIEW

File: pages/listings/listing/editListing.html
Comment (Jenfer93) [inline pages/listings/listing/editListing.html:16]: Maybe change the ID to something like editLisiting, so the same ID isn't used again :)

**Review by Jenfer93 — COMMENTED** [review]
Other than the one comment I think the form looks very good! Good job :) Wait for Åke to review it before any merge is done :)

File: pages/listings/listing/editListing.html
Comment (Gonlonge) [reply pages/listings/listing/editListing.html:16]: I'll fix it 👍

**Review by akeek — CHANGES_REQUESTED** [review]
All the tests I have written passes. But the edit and delete functions are identical. I think there should be som more input to the editSingleListing-function.
Also, there should be a method to each function? But the API-documentation have not provided that?

**Pull request closed without merging**
